conans/test/functional/command/remote_test.py

```diff
@@ -303,7 +303,7 @@ def verify_ssl_error_test(self):
         self.assertEqual(data["remotes"], [])
 
     def errors_test(self):
-        self.client.run("remote update origin url", assert_error=True)
+        self.client.run("remote update origin http://url", assert_error=True)
```

Comment:
Why this needs to change? Maybe it shouldn't.

Reply:
nope, I gonna revert

Reply:
done!